### PR TITLE
[FAST-531] Update Vulnerable Dependencies

### PR DESCRIPTION
**Pull request by ashimashrivastava**
## What does this PR do?

Fix vulnerable dependencies - https://github.com/pantheon-systems/eslint-config-pantheon/security/dependabot

## Resources

- [Jira ticket](https://getpantheon.atlassian.net/browse/FAST-531)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,26 +22,27 @@
   },
   "devDependencies": {
     "danger": "^11.2.3",
-    "eslint": "~8.9.0",
+    "eslint": "^8.50.0",
```
Comment (ckaethler) [inline package.json:25]: Why did we go down a version here?

File: package.json [hunk 2]
```diff
@@ -22,26 +22,27 @@
...
     "eslint-plugin-jest-dom": "~4.0.1",
     "eslint-plugin-jsx": "~0.1.0",
     "eslint-plugin-jsx-a11y": "~6.5.1",
     "eslint-plugin-prettier": "~4.0.0",
-    "eslint-plugin-react": "~7.28.0",
+    "eslint-plugin-react": "^7.33.2",
     "eslint-plugin-react-hooks": "~4.3.0",
-    "eslint-plugin-testing-library": "~5.0.5"
+    "eslint-plugin-testing-library": "~5.0.5",
+    "json5": "^2.2.3"
```
Comment (ckaethler) [inline package.json:45]: Why is this library being added?

File: package.json (hunk 1 above)
Comment (ashimashrivastava) [reply package.json:25]: Eslint undated itself in one of the dependencies that i was updating and wehn i checked online, 8.50.0 is the latest version - https://www.npmjs.com/package/eslint

File: package.json (hunk 2 above)
Comment (ashimashrivastava) [reply package.json:45]: I was fixing this - https://github.com/pantheon-systems/eslint-config-pantheon/security/dependabot/12 
which asked to Upgrade json5 to version 2.2.2 or later. And it ended up being added here. 
I actually wanted a review on this to make sure i did correct

File: package.json (hunk 1 above)
Comment (ckaethler) [reply package.json:25]: I read this as `8.5.0` instead of 8`8.50.0` 🤦 Simply a case of dyslexia rearing its pretty little head 🤣

**Comment by pantheon-circleci** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  :tickets: Jira ticket number seems to be missing from the PR title. Use format `[ADA-123]`. Your PM will thank you.
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 7bf5ffb00cb59f4e09aeaf5d552b90b4eac98700
</p>

File: package.json (hunk 2 above)
Comment (zzyou) [reply package.json:45]: You can remove this line, and make sure the dependencies of dependencies are pointing to the newly installed json5 in yarn.lock:
```suggestion
```

**Review by johndewolf — APPROVED** [review]

**Review by ckaethler — DISMISSED** [review]

**Review by zzyou — COMMENTED** [review]
Could you update its CHANGELOG?

**Review by BatyaFeldman — DISMISSED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -10,6 +10,14 @@ All notable changes to this project will be documented in this file. This projec
 
 ### Fixed/Updated
 
+## 1.0.7
+
```
[the remark this answers is not in the document]
Comment (ashimashrivastava) [reply CHANGELOG.md]: Cool! Will undo changelog changes and open a new pr for that. Thanks!

**Review by ckaethler — APPROVED** [review]

**Comment by zzyou** [comment]
We still need the item in CHANGELOG. We just don't add the new version here yet.

**Review by zzyou — APPROVED** [review]

**Review by drainpip — APPROVED** [review]

**Pull request merged**